### DISCUSSION TITLE
Splitting cue into wma

### DISCUSSION THREAD

**Discussion by RainerMathey** [enzo1982/freac#402] [Help & Support]
... i wanna split a cue / flac file into wma files, the process works fine, but the resulting wma-files are not tagged correctly and when i wanna tag them with mp3tag, there is an error on each file 🤔

**Comment by enzo1982** [comment]
Just tested this and the WMA files I create with fre:ac can be opened and tagged with Mp3tag without a problem. I tried all four WMA formats supported by fre:ac (WMA 9.2, WMA 9.2 Lossless, WMA Voice 9 and WMA 10 Pro).

Are you using an up to date version of Mp3tag? Also, what is wrong with the tagging in the first place?

**Reply by RainerMathey** [reply]
... i must say i'm under Arch Linux, i've tested the 1.16 from the chaotic-repo, the 1.16 windows version under wine & the appimage; none of this creates correctly tagged wma's and sometimes i cannot set different wma-profiles >>> if i tried all the same, but with for example lame mp3 as output eveything is fine >>> when you need some more info? i test

... and yes my mp3tag is the newest development build 😄

**Reply by enzo1982** [reply]
Oh, I was assuming you are on Windows if you are using WMA and Mp3tag...

WMA support on Linux is not very good, unfortunately. While fre:ac uses the official Microsoft APIs on Windows, on Linux it has to resort to FFmpeg for WMA. Also, there is no tagging support for WMA in fre:ac on Linux which explains your tagging issues.

The situation is not any better with Wine either. Wine does not implement the WMA codecs, so fre:ac's WMA support will not function correctly when using the Windows version under Wine. This is probably also the reason why Mp3tag cannot read the files, assuming you're running that under Wine too.

In the past it was possible to add WMA support to Wine by installing some Microsoft support package or the old Windows Media Player, but I'm not sure this still works.

I'll see if I can add basic tagging support for WMA on non-Windows systems in a future update.

**Reply by gsearle** [reply]
I just discovered this myself. I'm thinking of transitioning from Windows to Linux, but there are always bumps in the road. Thank you for fre:ac, by the way! I just converted some new MP3's to WMA's and was surprised to find the files were not tagged. Now I know why! I can tag the files in my music player on my phone as a workaround. Not ideal, but it will work.
